### ISSUE TITLE
[CAIA a11y Research - PILOT]: Ask VA - Desktop Pilot - Dashboard (September 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#92028]
> [!IMPORTANT]
> - **TICKET STATUS:** `SCHEDULED` ✅ 
> - **PILOT DATE:** 9/11/2024 11:00 a.m.
> - **TYPES OF ASSISTIVE TECH:** Desktop TTS
>     - A mix of [switch systems, Dragon, Voice Control (Apple), One-handed keyboards, a head mouse, or a sip and puff device](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Alternativenavigation).
>     - We will use MacOS Voice Control for this pilot.
> - **PROTOTYPE:** [Staging Link](https://staging.va.gov/contact-us/ask-va-too/)
> - **PILOT PREP NOTES:** [Notes](https://docs.google.com/document/d/14hFt_IItoGWyi4MaIuK3ru5qYNCeI4OObFseTw39z-0/edit#heading=h.7b53y4dqihzk)

# CAIA A11y Support
This is a child ticket of the research ticket [92003](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92003) - created to track work, completed from this **DESKTOP** pilot session.

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @coforma-jamie (pilot)
- @sara-amanda (ticket refinement & co-pilot)

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Ask VA
- OCTO-DE product owner: Becky Phung
- Product manager: CeeCee O'Connor
- Designer: Tyler Gindraux, Thomas Michaud
- FE engineer: Eddie Otero, Hemesh Patel
- Product/team Slack channel: #ask-va-public

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** Tuesday, September 11⋅11:00am – noon ET
-  **Prototype - Staging:** [Ask VA - Staging Link](https://staging.va.gov/contact-us/ask-va-too/)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Dashboard/Research%20plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%20research/07-2024%20Assistive%20tech/Dashboard/Conversation%20guide.md)
</details>

## Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

<details><summary>Toggle to view details about the 3 cohorts</summary>

### 12 Veterans to be Recruited
- minimum of 3 sessions with **screen reader** users
- minimum of 2 sessions with **alternative navigation** users
- minimum of 1 session with a **screen magnification** user

### Cohort 1: Screen reader users

We will recruit 6 Veterans who use screen readers to complete a minimum of 3 sessions. They should be daily users of screen readers. 3 people should have 1-4 years of experience and 3 people should have 5+ years of experience.

We'd like to recruit people who use a mix of different screen readers, such as, [NVDA, JAWS, VoiceOver (Apple) and Talkback (Android)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenreaders).

### Cohort 2: Alternative navigation users

We will recruit 4 Veterans who use alternative navigation to complete a minimum of 2 sessions. They should be daily users of alternative navigation tools.

We'd like to recruit people who use a mix of [switch systems, Dragon, Voice Control (Apple), One-handed keyboards, a head mouse, or a sip and puff device](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Alternativenavigation).

### Cohort 3: Screen magnification users

We will recruit 2 Veterans who use screen magnification to complete a minimum of 1 session. They should state that they always use screen magnification on VA.gov.

We'd like to recruit people who use a mix of different screen magnification software, such as, [ZoomText, ZoomText Fusion, Magnifier (Windows) and Zoom (Apple)](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Screenmagnification).

</details>

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y**
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA a11y Capacity Calendar:** Hold placed on calendar.
- [x] **Update CAIA a11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] Research Tracker Updated (Placeholder Info)
- [x] Research Tracker Updated (Scheduled Info)
- [ ] Research Tracker Updated (Post-Research Info)
- [x] **Review research** conversation guide
- [x] Request AT device and tech to be used in sessions
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot - _**CAIA and the team will determine what AT will be used in this session, specifically.**_
- [ ] **Review prototype** in `Staging`
- [x] Identify co-pilot (optional)
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: [Note Taking](https://docs.google.com/document/d/14hFt_IItoGWyi4MaIuK3ru5qYNCeI4OObFseTw39z-0/edit#heading=h.7b53y4dqihzk):** If there is only one a11y, notes cannot be taken due to pilot participation.
- [ ] [Additional Resource for Voice Control Testing](https://accessibility-manual.dwp.gov.uk/best-practice/voice-controller-testing)
```

**Comment by sara-amanda** [comment]
## Alternative Navigation
The Ask VA team is looking to use alternative navigation in this desktop pilot. They define alternative navigation as: 
> A mix of [switch systems, Dragon, Voice Control (Apple), One-handed keyboards, a head mouse, or a sip and puff device](https://depo-platform-documentation.scrollhelp.site/research-design/testing-your-prototype-with-assistive-technology-u#Testingyourprototypewithassistivetechnologyusers-Alternativenavigation).

**Comment by sara-amanda** [comment]
## ASK VA Availability

c/o [tygindraux](https://github.com/tygindraux) - Reference the [primary research ticket comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92003#issuecomment-2329857625
):

> Here are some times that work for us:
> 
> - Monday, September 9 between 11am - 4pm EST
> - Tuesday, September 10 between 11am - 4pm EST
> - Wednesday, September 11 between 11am - 4pm EST
> 
> Let me know if any of these work for you all 😊

**Comment by sara-amanda** [comment]
## Session Scheduled

- **Pilot**: Jamie
- **Co-Pilot**: Sara
- **Device**: Desktop
- **AT**: TTS
- **Invite on shared calendar**
- [Conversation guide](https://www.google.com/url?q=https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/design/User%2520research/07-2024%2520Assistive%2520tech/Dashboard/Conversation%2520guide.md&sa=D&source=calendar&ust=1726059749299248&usg=AOvVaw1DqVaUEVxCLnf2-gmsn57f)
- [Staging link](https://www.google.com/url?q=https://staging.va.gov/contact-us/ask-va-too/&sa=D&source=calendar&ust=1726059749299248&usg=AOvVaw1LizTZ7p-2pvXpH3ditRKa)
- CAIA to create notes document for pilot using template.

**Comment by coforma-jamie** [comment]
@sara-amanda made the [notes doc](https://docs.google.com/document/d/14hFt_IItoGWyi4MaIuK3ru5qYNCeI4OObFseTw39z-0/edit#heading=h.7b53y4dqihzk)!

**Comment by sara-amanda** [comment]
## Pilot Feedback from Session

**Feedback specific to:** Voice-Command Users

### Full screen share
- **Share the full screen** to see the `grid system` of numbers - or `accessible names`, depending on the user’s setting. 
    - Sharing the window, you can’t see the numbers the participant is using. 
    - To see the numbers or accessible names, you need to share the full screen. 

### Providing Feedback 
- **“Stop Listening”** should be spoken when providing feedback
    - If the user is talking to the researcher, they will need to tell the AT to “Stop Listening” in order to avoid interruptions. 

### Scannability and Consistency
- **Bold** `category` in card view. 
- **Bold** `last update` in card view. 
- This is for **consistency**, as it is bolded in the expanded view once you `click details`.
- This is also for **scannability**, when viewing the initial card. 

## Additional Questions

- **Will there be a `next page` button?** 
    - Can this be tested with a screen reader and voice command?
- **Can you have `multiple categories`?** 
- **Functionality Recommending:** 
    - Default to `ALL` - if you go filter by `category` or `status` it will filter to nothing.
